sdk/python/packages/flet-core/src/flet_core/page.py

```diff
@@ -2210,6 +2181,8 @@ def banner(self) -> Optional[Banner]:
     @banner.setter
     def banner(self, value: Optional[Banner]):
         self.__banner = value
+        if value is not None:
+            self.__controls.append(value)
```

Comment:
Shall we check that `value` is already in `self.__controls`?
...and I think deprecation notice should say "use `page.open(banner)` instead of adding to overlay?
The same applies to other popups.